lib/emoji.js

```diff
@@ -1,5 +1,6 @@
 /*jslint node: true*/
 require('string.prototype.codepointat');
+var toArray = require('lodash/toArray');
```

Comment:
I don't really want to pull in all of lodash as a dependency just for a way to convert something to an array - surely there must be a way to accomplish the same behaviour without lodash?